dev/tools/create_overlays.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+# TODO:
+#
+# - Check if the branch already exists in the remote => checkout
+# - Better error handling
+# - Just checkout, don't build
+# - Rebase functionality
+#
+
+set -x
+set -e
+set -o pipefail
+
+# setup_contrib_git("_build_ci/fiat", "https://github.com/ejgallego/fiat-core.git")
+setup_contrib_git() {
+
+    local _DIR=$1
+    local _GITURL=$2
+
+    ( cd $_DIR && \
```

Comment:
Remove `&& \`

Reply:
It doesn't need `set -e` to be "correct".